Dockerfile

```diff
@@ -47,15 +47,15 @@ ENV                 RELEASE=${release:+--release}
 # is rolled out, which no longer requires the `protoc`, we'll be able to
 # get rid of this.
 RUN                 apt-get update \
-    && apt-get install -y --no-install-recommends protobuf-compiler lld llvm
+    && apt-get install -y --no-install-recommends protobuf-compiler 
 
 WORKDIR             /app
 # Cache dependancies
 COPY --from=plan    /app/recipe.json recipe.json
-RUN                 RUSTFLAGS="-C link-arg=-fuse-ld=lld" cargo chef cook --recipe-path recipe.json ${RELEASE}
+RUN                 cargo chef cook --recipe-path recipe.json ${RELEASE} --analytics analytics
```

Comment:
Whats `--analytics analytics`?

Reply:
Lol this was supposed to be feature, must've renamed it by misate
Removed as we got rid of feature and moved analytics into main code